AdyenActions/UI/View Controllers/QR Code/QRCodeView.swift

```diff
@@ -0,0 +1,118 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+import UIKit
+
+internal protocol QRCodeViewDelegate: class {
+
+    func copyToPasteboard()
+}
+
+internal final class QRCodeView: UIView, Localizable {
+    
+    private let model: Model
+    
+    /// The delegate of the view
+    internal weak var delegate: QRCodeViewDelegate?
+    
+    // :nodoc:
+    public var localizationParameters: LocalizationParameters?
+    
+    internal init(model: Model) {
+        self.model = model
+        super.init(frame: .zero)
+        buildUI()
+        backgroundColor = model.style.backgroundColor
+    }
+    
+    @available(*, unavailable)
+    internal required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func buildUI() {
+        addLogo()
+        addInstructionLabel()
+        addCopyButton()
+    }
+    
+    internal override func layoutSubviews() {
+        super.layoutSubviews()
+        
+        copyButton.adyen.round(using: model.style.copyButton.cornerRounding)
+    }
+    
+    private func addInstructionLabel() {
+        addSubview(instructionLabel)
+        instructionLabel.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            instructionLabel.topAnchor.constraint(equalTo: logo.bottomAnchor, constant: 24),
+            instructionLabel.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 37),
+            instructionLabel.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -37.0)
+        ])
+    }
+    
+    private func addLogo() {
+        addSubview(logo)
+        logo.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            logo.bottomAnchor.constraint(equalTo: topAnchor, constant: 10),
+            logo.centerXAnchor.constraint(equalTo: centerXAnchor)
+        ])
+    }
+    
+    private func addCopyButton() {
+        addSubview(copyButton)
+        copyButton.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            copyButton.topAnchor.constraint(equalTo: instructionLabel.bottomAnchor, constant: 29),
+            copyButton.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 16),
+            copyButton.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -16),
+            copyButton.bottomAnchor.constraint(equalTo: safeAreaLayoutGuide.bottomAnchor, constant: -18)
+        ])
+    }
+    
+    private lazy var copyButton: SubmitButton = {
+        let accessiblityIdentifier = ViewIdentifierBuilder.build(scopeInstance: "adyen.QRCode", postfix: "copyButton")
```

Comment:
Usually you can use `ViewIdentifierBuilder.build(scopeInstance: self, postfix: "BLABLA")` - it will use self's identifier + prefix.

Reply:
that would mean i need to set accessibility identifier for the `QRCodeView` instance somewhere, right?

Reply:
Yes. However if this is standalone view - sure, you need some "base" for identifier.